package.json

```diff
@@ -47,7 +47,9 @@
   "dependencies": {
     "chalk": "^4.0.0",
     "yargs": "^15.3.1",
-    "@kentico/kontent-management": "~0.3.19"
+    "@kentico/kontent-management": "~0.3.19",
+    "@kentico/kontent-backup-manager": "~1.2.0",
```

Comment:
All scripts are run silently, meaning that when you e.g. export / import / clean large projects you see nothing in console other then "Starting backup tool". It would be nice to let users know the progress and let them know what is going on (and that the tool is not stucked). 

You setup logging using `onExport`, `onImport` and `onDelete` callbacks in configuration of each service. Having a configuration key to disable logging could be there too. By default I would however display logging.